src/kOS.Safe/Compilation/KS/kRISC.tpg

```diff
@@ -141,7 +152,8 @@ remove_stmt    -> REMOVE expr EOI;
 log_stmt       -> LOG expr TO expr EOI;
 break_stmt     -> BREAK EOI;
 preserve_stmt  -> PRESERVE EOI;
-declare_stmt   -> DECLARE PARAMETER? IDENTIFIER (COMMA IDENTIFIER)* EOI;
+declare_stmt   -> DECLARE ( (PARAMETER IDENTIFIER (COMMA IDENTIFIER)* EOI) | (FUNCTION IDENTIFIER instruction_block EOI?) | (IDENTIFIER TO expr) EOI);
```

Comment:
If people decide they hate the ```DECLARE foo TO val ``` syntax, we can always add an optional token "INIT" here.  That would be easy.  Thus it would support saying ```DECLARE foo INIT TO val.```.